### ISSUE TITLE
Use r-universe as a possible source

### ISSUE THREAD

**Issue by Bisaloo** [r-lib/available#71]
I believe it is important to offer the possibility to check for possible collisions with other r-universes since this might be a problem for users that enabled a lot of universes.

**Comment by jimhester** [comment]
I think this would happen automatically if you have them in your `options(repos)` when you run available.

**Comment by Bisaloo** [comment]
Good idea but I checked and no, it doesn't happen.

This is because `available_on_cran()` uses `default_cran_repos` as a default:

https://github.com/r-lib/available/blob/8df8baf0b83d1f2ef9464f159dc925c67c7434b3/R/cran.R#L7

https://github.com/r-lib/available/blob/8df8baf0b83d1f2ef9464f159dc925c67c7434b3/R/cran.R#L38-L40

So to get them with `available()`, or `available_on_cran()`, one would have to explicitly specify the universe URL in `repos`. E.g.:

```r
available::available("semaforos", repos = "https://bisaloo.r-universe.dev")
```

You might consider this is the desirable behaviour though. I don't know :shrug:

**Comment by jimhester** [comment]
Yeah, I think the default behavior is probably fine in this case. It could be worthwhile to add an example showing how to use it with an r-universe though.

Would you be interested in doing a pull request to add the example?
